src/Articulate/Articulate.csproj

```diff
@@ -96,7 +96,7 @@
     <PackageReference Include="Umbraco.Cms.Core" Version="10.1.0" />
     <PackageReference Include="System.ServiceModel.Syndication" Version="5.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection.Abstractions" Version="6.0.14" />
-    <PackageReference Include="WilderMinds.MetaWeblog" Version="5.1.0" />
+    <PackageReference Include="WilderMinds.MetaWeblog" Version="5.0.0" />
```

Comment:
I'm a little confused. Why does this need to be changed and what does this have to do with Umbraco.Forms?

Reply:
The current 5.0.0 branch will not install/update into a v10.4 site with Umbraco.Forms 10.3 installed, due to a package installation error.

Umbraco.Forms (or one of it's dependencies) has a constraint on `Microsoft.Extensions.DependencyInjection.Abstractions (>= 6.0.0 && < 7.0.0)`

WilderMinds.MetaWeblog 5.1 has a constraint on [Microsoft.Extensions.DependencyInjection.Abstractions](https://www.nuget.org/packages/WilderMinds.MetaWeblog/#dependencies-body-tab) (>= 7.0.0) 

Reverting from 5.1 to 5.0 resolves issue, unsure if other workaround.

Reply:
That's unfortunate. This is a bug with both of those packages then. Umbraco Forms should restrict to less than 7 (AFAIK there are no breaking changes there) and WilderMinds.MetaWeblog shouldn't have a min of 7 since I'm sure they are not using anything explicitly in 7.

Reply:
Have posted an issue here https://github.com/shawnwildermuth/MetaWeblog/issues/17